tests/consensus/test_soft_voided.py

```diff
@@ -19,7 +19,7 @@ def assertNoParentsAreSoftVoided(self, tx):
             self.assertNotIn(settings.SOFT_VOIDED_ID, tx2_voided_by)
 
     def test_soft_voided(self):
-        txA_hash = bytes.fromhex('4586c5428e8d666ea59684c1cd9286d2b9d9e89b4939207db47412eeaabc48b2')
+        txA_hash = bytes.fromhex('1ae4ce163495279dafddca041b7c99abd71af55f29568746f3a20deead15f14d')
```

Comment:
The pdfs attached to this PR do not have the labels txA. Why is that? Did you disable it?

Reply:
You're right. I think I may have added partial pdfs (which I generated in the middle of the test) instead of full pdfs.

I'll generate new ones that are full, for both the "before" and "after" and update in the description

Reply:
Thanks!